GameProject/Assets/TurnBasedAssets/Scripts/MouseSelection.cs

```diff
@@ -5,11 +5,11 @@ namespace TurnBasedAssets.Scripts
     public class MouseSelection : MonoBehaviour
     {
         private ISelection _selection;
-        [SerializeField] GameObject cube;
-        [SerializeField] float distanceY;
-        [SerializeField] float offset;
-        [SerializeField] float gridSize;
-        [SerializeField] Camera camera;
+        [SerializeField] public GameObject cube;
```

Comment:
public fields should be reworked. Also you have `SerializeFieldAttribute` on multiple public fields. Please fix.